rfc_process.md

```diff
@@ -0,0 +1,85 @@
+# Versioning Process for the OpenTracing Specification
+The OpenTracing specification consists of a cross-language, abstract interface, along with an attendant set of language-specific interfaces. Every interface is versioned, with each language interface supporting a version of the abstract interface.
+
+In order effect changes to the OpenTracing specification, all interfaces are developed together in a three step process:
+
+* DRAFT: An RFC is drafted to define and justify the desired changes.
+* TEST: A new version of each language interface is designed, tested, and released.
+* ACCEPTED: The abstract interface is versioned to reflect the final changes.
+
+# Proposal Components
+
+## RFC (Request for Change)
```

Comment:
Usually, "RFC" is a "request for comments", not change. Did you really mean "change"?